.github/ISSUE_TEMPLATE.md

```diff
@@ -18,4 +18,7 @@ STOP!! Please don't raise security vulnerabilities here. Head over to https://pi
 🎁 Enhancement
 Please start by describing the problem that you are trying to solve. There may already
 be a solution, or there may be a way to solve it that you hadn't considered.
+
+- Managed dependency upgrade
+You DO NOT need to raise a minor version upgrade issue for a managed dependency as there's a semi-automatic process for checking managed dependencies for new minor versions before a release.
```

Comment:
I wonder if insisting on "minor" is the right call here. bomr is also able to let us know about major upgrades, right @wilkinsona ?

Reply:
Right. If a change just modifies the version property in `spring-boot-dependencies` it's quicker and easier for Bomr to take care of it, irrespective of it's a new major, minor, or patch release. Pull requests for upgrades that are more involved (such as [this one](https://github.com/spring-projects/spring-boot/pull/13322)) are still most welcome, irrespective of whether it's a new major, minor or, hopefully not, patch release.

Reply:
Looking at @wilkinsona 's comment I realized I used the wrong term. "minor" meant to be "patch" as I thought it could handle only "patch" versions.

Based on the @wilkinsona 's comment I'd like to change the section to:

> You DO NOT need to raise an issue for a managed dependency version upgrade as there's a semi-automatic process for checking managed dependencies for new versions before a release. But pull requests for upgrades that are more involved than just a version property change are still most welcome.

WDYT?

Reply:
That sounds good to me Johnny, thank you!

Reply:
@snicoll Thanks for the quick feedback! I updated as suggested.